### PR TITLE
Asset note model

### PR DESCRIPTION
**Pull request by igorsergiichuk**
# Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem

AssetNote model was absent

## How I resolved it

Added AssetNote model

## How to verify it

Test it!

Notify the following people: @darrynten

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,6 +295,7 @@ Models marked with an asterix are pure CRUD models
     - [x] Account Receipt *
   - [x] Analysis Category
   - [x] Analysis Type
+  - [x] Asset Note *
```
Comment (darrynten) [inline README.md:298]: See #73 about removing it from further down the README

File: tests/SageOne/Models/AssetNoteModelTest.php [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            ],
+        ]);
+    }
+
+    public function testInject()
+    {
+        $this->verifyInject(AssetNote::class, function ($model) {
+            $this->assertEquals($model->assetId, 1);
+            $this->assertEquals($model->id, 2);
+            $this->assertEquals($model->subject, "sample string 3");
```
Comment (darrynten) [inline tests/SageOne/Models/AssetNoteModelTest.php]: These should be single quoted

File: tests/SageOne/Models/AssetNoteModelTest.php [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        $this->verifyGetAll(AssetNote::class, function ($results) {
+            $this->assertCount(2, $results);
+
+            $this->assertInstanceOf(AssetNote::class, $results[0]);
+            $this->assertInstanceOf(AssetNote::class, $results[1]);
+
+            $model1 = $results[0];
+            $this->assertEquals($model1->assetId, 1);
+            $this->assertEquals($model1->id, 2);
+            $this->assertEquals($model1->subject, "sample string 3");
```
Comment (darrynten) [inline tests/SageOne/Models/AssetNoteModelTest.php]: Single quotes

File: tests/SageOne/Models/AssetNoteModelTest.php [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    {
+        $this->verifyDelete(AssetNote::class, 2, false);
+    }
+
+    public function testSave()
+    {
+        $this->verifySave(AssetNote::class, function ($model) {
+            $model->assetId = 1;
+            $model->id = 2;
+            $model->subject = "sample string 3";
```
Comment (darrynten) [inline tests/SageOne/Models/AssetNoteModelTest.php]: Some more single quotes

**Review by darrynten — CHANGES_REQUESTED** [review]
Some minor change requests

File: tests/SageOne/Models/AssetNoteModelTest.php (hunk 2 above)
Comment (igorsergiichuk) [reply tests/SageOne/Models/AssetNoteModelTest.php]: done

File: tests/SageOne/Models/AssetNoteModelTest.php (hunk 3 above)
Comment (igorsergiichuk) [reply tests/SageOne/Models/AssetNoteModelTest.php]: done

File: tests/SageOne/Models/AssetNoteModelTest.php (hunk 4 above)
Comment (igorsergiichuk) [reply tests/SageOne/Models/AssetNoteModelTest.php]: done

**Review by darrynten — APPROVED** [review]

**Pull request merged**